### ISSUE TITLE
Document local installation of Appsody CLI for developers

### ISSUE THREAD

**Issue by bschrammIBM** [kabanero-io/docs#38]
This is the Appsody CLI, it sets the default hub url to the Kabanero public hub. The hub can be changed to point to Champ’s hub (or manually use the CLI to change it).
There are current instructions on the Kabanero.io website: 
https://kabanero.io/developer/cli/

There are new instructions included in the proposed Getting Started flow.
These instructions need to be reviewed and updated.

**Comment by jantley-ibm** [comment]
Dev: David Harris ID: Helen Masters

**Comment by helenmasters** [comment]
@bschrammIBM is this task a request for a docs new page or a request to update the existing https://kabanero.io/developer/cli/ page?

Please can you also point me to the "new instructions" which you refer to in the "proposed Getting Started flow"?

**Comment by SueChaplain** [comment]
@helenmasters - if this work is to update the new getting started page (see https://github.com/kabanero-io/kabanero-website/issues/158), I don't think this is a huge amount of work. Let's discuss. I'm happy to own it if you are not going to be able to cover it or assign to someone else locally)

**Comment by bschrammIBM** [comment]
Yes It is to edit and polish the instructions for installing the Appsody CLI. One of the issues is the installation instructions are Mac-specific. Should we include instructiosn for PC too?
So to edit and embellish the instructions on tthis page https://kabanero-kdsee.us-south.cf.test.appdomain.cloud/developer-tools/

```
The Kabanero uses Appsody as its CLI experience to access collections

Install XCode Command Line Tools and Homebrew.

xcode-select --install
To install Homebrew run:

/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
Install the Appsody plugin.

brew install appsody/appsody/appsody
Select the Kabanero Collection.

To see a list of all available public collections run:

$ appsody list
Need more help using Appsody? Learn More 
```

**Comment by SueChaplain** [comment]
We've got 4 different platforms for Appsody :-/ macOS is probably the slickest if we are trying to make it look simple, so we could just link to the Appsody install topic for other platforms as a footnote?

I noticed that the VS Code install gets as far as initializing a collection (although the screen shot is missing the Appsody stacks), so we probably need to do more work to fix and balance the content a little better across these 3 tools.

**Comment by SueChaplain** [comment]
@kSee04 - did your PR https://github.com/kabanero-io/kabanero-website/pull/161/files change this content already? It looks like it but the URL I have (https://kabanero-kdsee.us-south.cf.test.appdomain.cloud/developer-tools/) doesn't refresh ... maybe that's a stale site? Where can I preview the latest changes for the Getting Started please?

**Comment by alohr51** [comment]
@SueChaplain we have a couple of test instances you can see the above changes on https://kabanero.us-south.cf.test.appdomain.cloud/developer-tools/

**Comment by SueChaplain** [comment]
@alohr51 we are trying to go through the open issues and close any out that are complete. We believe this one is done but perhaps held back with the Getting Started section. Is there a preview of this work to double check please? The one provided in an earlier comment is no longer live.

Any idea when Getting Started will make it to production? Thanks!

**Comment by marikaj123** [comment]
@alohr51 - Hi Andrew, any status update on this issue would be appreciated.

**Comment by alohr51** [comment]
so looks like this is blocked on kabanero-io/kabanero-website/issues/193 which involves getting the new content from Alan and Nate for the dev tools section of the new experience.

Although, if design (cc @mdenby) decided this could be a doc/guide instead of on a page in the website, then doc team could probably go ahead and create that install doc/guide and be unblocked. Then the website could link to the guide when it talks about the appsody cli

**Comment by SueChaplain** [comment]
Is this still a valid issue?

>so looks like this is blocked on kabanero-io/kabanero-website/issues/193 which involves getting the new content from Alan and Nate for the dev tools section of the new experience.

Might it not be superseded by the content coming from Alan and Nate? If not, we need visibility of the work that's underway in order to contribute some changes. If there is no change in status by 14th October this issue is unlikely to make the 18th October date.

**Comment by mingcyu** [comment]
Need to discuss with Alan whether this is still needed given the page is being replaced

**Comment by mingcyu** [comment]
@ralanlittle Alan, is this issue still needed given that it will be replaced per discussion above?

**Comment by mingcyu** [comment]
Closing issue as no strong opinion. Please reopen if needed
